flake8.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+# There are 3 levels: F (FATAL), E (ERRORS) and W(WARN)
+# Everything is blacklisted here only due to excessive hits in existing code
+# base. The ignores should be removed in the severity level (e.g. F first, then E, then W)
+find -type f -name "*.py" -print | grep -v abichecker | grep -v openqa | \
```

Comment:
Ideally, this file should allow a git reference to be passed (which travis can provide) and use the git diff to only check the files that were changed. I suppose if the check runs fast and we end up cleaning everything this approach also works, but seems like it avoid you having to fix all the issue. **If the pip8 has the ability to run on diffs then we can fix things whenever the line is changed, which would be the real preference.**

Reply:
I see where this is going, but I would prefer to run it locally on a list of files. As a background, i have that integrated in my editor and it is difficult to even save a file of it doesn't pass.

None of those checks are difficult or time-consuming to fix and it just takes about a second to run over all of what is in git.

Reply:
In one of the projects that I am involved in we're doing that increments code style fixing and it causes every pull request to become bloated with the change plus any style fixes necessary to pass testing and it makes every review harder because of both of it mangled in one place, *and* it is frustrating for the submitter because of the need to care about a bunch of unrelated stuff.

Reply:
The script could easily work in both forms, but we can keep it this way. Still not sure I like the level of reformatting in the original PR.